rdrf/rdrf/management/commands/create_longitudinal_followup_entries.py

```diff
@@ -14,21 +14,28 @@ def add_arguments(self, parser):
         parser.add_argument("--limit", type=int, default=None)
         parser.add_argument("--allow-duplicates", action="store_true")
         parser.add_argument("--check-for-form-response", action="store_true")
+        parser.add_argument("--require-consents", type=str, nargs="+", default=[])
+        parser.add_argument("--require-living", action="store_true")
 
     def handle(self, *args, **options):
         registry_code = options["registry_code"]
         longitudinal_followup_name = options["longitudinal_followup_name"]
         limit = options["limit"]
         allow_duplicates = options["allow_duplicates"]
         check_for_form_response = options["check_for_form_response"]
+        require_consents = options["require_consents"]
+        require_living = options["require_living"]
 
         try:
             registry = Registry.objects.get(code=registry_code)
             longitudinal_followup = LongitudinalFollowup.objects.get(name=longitudinal_followup_name)
+            consent_questions = [ConsentQuestion.objects.get(code=code) for code in require_consents]
         except Registry.DoesNotExist:
             raise CommandError(f"Registry {registry_code} does not exist")
         except LongitudinalFollowup.DoesNotExist:
             raise CommandError(f"LongitudinalFollowup {longitudinal_followup_name} does not exist")
+        except ConsentQuestion.DoesNotExist:
+            raise CommandError(f"ConsentQuestion does not exist")
```

Comment:
Just needs the consent question code in the exception string, for consistency with how the other `DoesNotExist` exceptions are handled.